### ISSUE TITLE
Buttons don't change to Windows 10 style

### ISSUE THREAD

**Issue by petrikos7** [valinet/ExplorerPatcher#3910]
Hi. After the latest update taskbar buttons cannot change to Windows 10 style. Any solutions?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.  And what language your windows is in.
Please post explorer patcher version.

Where in the EP Properties application are you changing the taskbar buttons and which taskbar buttons are you changing?

**Comment by Amrsatrio** [comment]
Install latest 67.1. Use EP's properties GUI > Taskbar section > click on the text at the right of Taskbar style > select Windows 10 (ExplorerPatcher). There are no known issues at the moment that would prevent you from changing the taskbar to Windows 10 on 21H2/22H2/23H2 or Windows 10 (ExplorerPatcher) on 22H2/23H2/24H2, otherwise I would've hurried to fix it.

**Comment by petrikos7** [comment]
Hello, I am using Windows 11 23H2 22631.4317 version 64 Greek and the latest version 67.1. I am using Explorer Patcher for about 2 years without any problems. EP Properties -> Taskbar -> Taskbar style

**Comment by pyrates999** [comment]
In windows 11 24h2, the code for the original windows 10 taskbar was removed.  So that's why you need to set the taskbar value to use the EP implemented version of windows 10 taskbar.

**Comment by pyrates999** [comment]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

please see this for more info: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Comment by petrikos7** [comment]
You are right. Let me rephrase: The "Windows 10 (ExplorerPatcher)" option is available. But when I select it, the taskbar remains in Windows 11 mode.

**Comment by Amrsatrio** [comment]
Have you restarted explorer after making the change? Do you have a third party antivirus? Have you tried reinstalling EP 67.1? Do you have any other Windows modding software like StartAllBack or Windhawk?

**Comment by petrikos7** [comment]
I just downloaded and reinstalled EP 67.1, restarted explorer but taskbar still in Windows 11 mode. No third party antivirus nor any modding software.

**Comment by Amrsatrio** [comment]
Show a full screenshot showing your EP properties window in the Taskbar section and the taskbar itself

**Comment by petrikos7** [comment]
![2024-12-12_18h55_30](https://github.com/user-attachments/assets/3e994ace-34c2-4cd7-8470-bf22f0356502)

**Comment by Amrsatrio** [comment]
It is already the Windows 10 style taskbar... the buttons too... not sure what you want then

**Comment by petrikos7** [comment]
Ok, that's embarrasing! Thank you. That is what I wanted! Thank you again for helping and sorry for misunderstanding the situation.
